### ISSUE TITLE
Use Keras to implement an LSTM

### ISSUE THREAD

**Issue by kaiyamag** [kaiyamag/emotion-detection#5]
The model takes a FastText embedding as input and outputs a percent prediction for all 27 GoEmotions classes.

**Comment by kaiyamag** [comment]
The Model class demonstrates a working LSTM, albeit with low accuracy. There are functions to build the training, validation, and test datasets, initialize and compile the LSTM Keras model, train the model, test the model by making predictions on test data, print a preliminary confusion matrix, and print a prediction for an individual string.

Next steps include adding F1 score evaluation (which involves defining the threshold at which an output vector is 'correct,' which will also be used to improve the relevancy of the confusion matrix), updating unit test cases for dataset processing, and verifying that the training dataset is built as expected. For the sake of testing, one epoch, a low number of comments and limited FastText dictionary are being used for training.
